.travis.yml

```diff
@@ -51,6 +51,19 @@ stage_linux: &stage_linux
   language: python
   python: 3.5
 
+stage_linux_latex: &stage_linux_latex
+  <<: *stage_linux
+  before_install:
+    - sudo apt-get install -y texlive-latex-base
+    - sudo apt-get install -y texlive-fonts-recommended
+    - sudo apt-get install -y texlive-fonts-extra
+    - sudo apt-get install -y texlive-latex-extra
+    - sudo apt-get install -y poppler-utils
+    - sudo apt-get install -y texlive-pictures
```

Comment:
Can't we do these as a single line? That should be marginally faster since we don't have to re call apt-get 6 times. Also why don't we just do this using travis packages:

https://docs.travis-ci.com/user/installing-dependencies/#adding-apt-packages

So replace L56-62 with:

```
addons:
  apt:
    packages:
    - texlive-latex-base
    - texlive-fonts-recommended
    - texlive-fonts-extra
    - texlive-latex-extra
    - poppler-utils
    - texlive-pictures
```